packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -28,9 +32,10 @@ export const ButtonView: IButtonComponent['view'] = props => {
       {...buttonProps}
       aria-disabled={disabled}
     >
-      <Slots.stack horizontal as="span" gap={8} verticalAlign="center" horizontalAlign="center">
-        {icon && <Slots.icon />}
-        {content && <Slots.content />}
+      <Slots.stack horizontal as="span" gap={8} verticalAlign="center" horizontalAlign="center" verticalFill>
+        <Slots.icon />
+        <Slots.content />
+        {renderTestButton && <Slots.button />}
         {children}
         {Menu && (
```

Comment:
Is it possible to render the Menu "NEXT" to the button now using fragments? The issue is that if you decide to render the menu inline (not in a layer) this creates buttons inside of buttons for no reason.

Reply:
Added as a work item on that issue.